<!--AUTHOR: Neset Unver Akmandor (NUA)-->
<!--E-MAIL: akmandor.n@northeastern.edu-->

<launch>
  <!-- CONFIGURATION OPTIONS
    config_scan_utility
  -->
  <arg name="config_scan_utility"  default="config_scan_utility" />

  <!-- PUBLISH CONFIGURATION PARAMETERS -->
  <rosparam file="$(find mobiman_simulation)/config/$(arg config_scan_utility).yaml"/>

  <node name="scan_utility_server" pkg="mobiman_simulation" type="scan_utility_server" output="screen" />
</launch>